wp-content/themes/pub/wporg-learn-2020/functions.php

```diff
@@ -244,3 +244,27 @@ function wporg_get_workshop_presenters( $workshop = null ) {
 
 	return $wp_users;
 }
+
+/**             
+ * Display a featured image, falling back to the VideoPress thumbnail if no featured image was explicitly set.
+ *          
+ * @param $post The Workshop post for which we want the thumbnail.
+ * @param $size The image size: 'medium', 'full'.
+ */     
+function wporg_get_post_thumbnail( $post, $size = 'post-thumbnail' ) {
```

Comment:
I'd be tempted to just rely upon the global `$post` object here, skip the arg and do `$post = get_post()` in the function instead